NEWS

```diff
@@ -8,6 +8,9 @@ v1.4.3rc1
 - Added proposed OpenSHMEM wait/test-any/some/all to shmemx interfaces.
 - Added proposed OpenSHMEM nonblocking fetching atomic operations to shmemx
   interfaces.
+- Added proposed OpenSHMEM SHMEM_CTX_INVALID constant to shmemx interfaces.
```

Comment:
Should we say `SHMEMX_CTX_INVALID`? Although we did mention as shmemx interface. Both ok, I guess.

Reply:
Yeah, I went back and forth on that, too.

Reply:
:man_shrugging:

Reply:
I am ok then with whatever you decide :)